plugins/org.jboss.reddeer.swt/src/org/jboss/reddeer/swt/impl/text/LabeledText.java

```diff
@@ -37,6 +38,6 @@ public LabeledText(String label) {
 	 * @param label the label
 	 */
 	public LabeledText(ReferencedComposite referencedComposite, String label) {
-		super(referencedComposite, 0, new WithLabelMatcher(label));
+		super(referencedComposite, 0, new WithLabelMatcher(new RegexMatcher(label + "\\s*\\**")));
```

Comment:
hmm... interesting point... I will also add a test for such case and will see what I can do